### PR TITLE
Feature/tropess co

### PR DESCRIPTION
**Pull request by jeromebarre**
## Description

This is the new converter for the TROPESS CO CrIS Products

### Issue(s) addressed

https://github.com/JCSDA-internal/AOP22/issues/193
https://github.com/jcsda-internal/aop22/issues/199

## Dependencies

corresponding ufo and ufo data PR to be issued

### CODE REVIEW

**Comment by BenjaminRuston** [comment]
this looks like it's getting close @jeromebarre we are going to move ahead with the Common calling structure PR1087 so would be great to get this in before that goes

File: src/compo/tropess_co_nc2ioda.py [hunk 1]
```diff
@@ -258,24 +255,22 @@ def make_obsVars(self):
...
-        self.obsvars= obsvars
+        self.obsvars = obsvars
 
     def make_AttrData(self):
         """
         Make a dictionary of AttrData based on obsvars that is created using
         the levels specified by the user (command line args).
         """
-        AttrData= {
+        AttrDatai = {
```
Comment (mer-a-o) [inline src/compo/tropess_co_nc2ioda.py:267]: is `i` a typo?

Comment (jeromebarre) [reply src/compo/tropess_co_nc2ioda.py:267]: yes it is a vi typo...

**Comment by BenjaminRuston** [comment]
@jeromebarre and @mer-a-o thanks! went ahead and promoted this from draft to a real honest to Jehoshaphat PR,, maybe we will each get a badge for taking this to fruition

**Comment by jeromebarre** [comment]
this was draft pr... it's not ready can we keep it as draft for now. thanks

**Comment by BenjaminRuston** [comment]
sure thing @jeromebarre converted back to draft

**Comment by jeromebarre** [comment]
Tests with intel and gnu are passing for me on discover. I haven't tested on orion and s4. Not sure what's going on AWS CodeBuild us-east-1 (ioda-conventers-internal-intel)?

**Comment by jeromebarre** [comment]
@BenjaminRuston if you are ok with this https://github.com/JCSDA-internal/ioda-converters/pull/1150/commits/1b47f565b1f2ab123e3560b8094956de373a2a70 then we are good to go

**Review by mer-a-o — APPROVED** [review]

**Review by PatNichols — APPROVED** [review]

**Pull request merged**